sensor/kubernetes/certrefresh/securedcluster_tls_issuer_test.go

```diff
@@ -272,7 +272,7 @@ func (s *securedClusterTLSIssuerTests) TestSecuredClusterTLSIssuerProcessMessage
 	fixture.tlsIssuer.Notify(common.SensorComponentEventCentralReachable)
 	s.Require().NoError(fixture.tlsIssuer.Start())
 
-	assert.NoError(s.T(), fixture.tlsIssuer.ProcessMessage(msg))
+	assert.NoError(s.T(), fixture.tlsIssuer.ProcessMessage(context.TODO(), msg))
```

Comment:
```suggestion
	assert.NoError(s.T(), fixture.tlsIssuer.ProcessMessage(s.T().Context(), msg))
```
This should apply to the other `ProcessMessage` calls in this test file.